overviews/quasiquotes/expression-details.md

```diff
@@ -565,7 +557,7 @@ is critical:
 
 ## For and For-Yield Loops
 
-For and For-Yield expressions allow to write monadic style comprehensions that desugar into calls to `map`, `flatMap`, `foreach` and `withFilter` methods:
+For and For-Yield expressions allow us to write a monadic style comprehension that desugar into calls to `map`, `flatMap`, `foreach` and `withFilter` methods:
```

Comment:
For and For-Yield should probably be backquoted too for consistency with val and var.